packages/frontend-shared/cypress/support/commands.ts

```diff
@@ -4,11 +4,11 @@ import urql, { TypedDocumentNode, useQuery } from '@urql/vue'
 import { print, FragmentDefinitionNode } from 'graphql'
 import { testUrqlClient } from '@packages/frontend-shared/src/graphql/testUrqlClient'
 import { Component, computed, watch, defineComponent, h } from 'vue'
-
 import { ClientTestContext } from '../../src/graphql/ClientTestContext'
 import type { TestSourceTypeLookup } from '@packages/graphql/src/testing/testUnionType'
 import { createI18n } from '@packages/launchpad/src/locales/i18n'
 import 'cypress-file-upload'
+import { createRouter } from '@packages/app/src/router/router'
```

Comment:
We should not be imported from `app` or `launchpad` into `frontend-shared` - this makes `launchpad` and `app` coupled now, and their concerns bleed into each other (eg, launchpad does not use a router, but now it has one). No shared packages should be importing from other outside packages.

I am not sure how, but we need some way to customize the `cy.mount` command. I'm not really sure there's a compelling reason to have `cy.mount` at all (or at least, to share it). 

We could either:

- don't even have a `cy.mount`, but just have a function we import - eg `import { mount } from '../some/util'.
- find some way to easily customize `cy.mount` on a package-by-package basis

Reply:
> We should not be imported from `app` or `launchpad` into `frontend-shared` - this makes `launchpad` and `app` coupled now, and their concerns bleed into each other (eg, launchpad does not use a router, but now it has one). No shared packages should be importing from other outside packages (eg, `runner-shared` does not import from `runner`, etc).

Agreed. Because `useI18n` was already here, I think it'll be fine for now (at least until we refactor it, like below)

> I am not sure how, but we need some way to customize the `cy.mount` command. I'm not really sure there's a compelling reason to have `cy.mount` at all (or at least, to share it).
> 
> We could either:
> 
> * don't even have a `cy.mount`, but just have a function we import - eg `import { mount } from '../some/util'.

Yeah that was my thought, too.

> * find some way to easily customize `cy.mount` on a package-by-package basis